META.json

```diff
@@ -1,15 +1,17 @@
 {
    "name": "currency",
    "abstract": "Enumerable currency list, stored in a single-byte, fixed-length type.",
-   "version": "0.0.3",
-   "maintainer": [ "Chris Travers <chris.travers@adjust.com>", "Manuel Kniep <rapimo@adjust.com>" ],
+   "version": "0.0.4",
+   "maintainer" : [
+      "adjustgmbh"
+   ],
    "license": {
       "PostgreSQL": "http://www.postgresql.org/about/licence"
    },
    "provides": { 
       "currency": {
          "file": "sql/pg-currency.sql",
-         "version": "0.0.3",
+         "version": "0.0.4",
```

Comment:
The current extensions version is `0.0.3`:
https://github.com/adjust/pg-currency/blob/master/currency.control#L3

I guess it will be necessary to dump its version and create new migration script.